src/renderer/wg_engine/tvgWgShaderSrc.cpp

```diff
@@ -281,7 +281,10 @@ struct Matrix {
 
 // ColorInfo
 struct ColorInfo {
-    color: vec4f
+    format: u32,
+    dummy0: f32,
```

Comment:
Question: what are these dummies for?

Reply:
> Question: what are these dummies for?

There are 2 reasons:
1. Its reserved values for future functionality, f.e. UV flipping
2. Its alligmant for uniform data structures. The most effective data structure should have size alligned to 32x4 bits. Thats why we have 4 values by 4 bytes and two of them are dummy for now